azure/application-gateway/advanced/examples/advanced/main.tf

```diff
@@ -0,0 +1,151 @@
+provider "azurerm" {
+  features {
+    resource_group {
+      prevent_deletion_if_contains_resources = false
+    }
+  }
+}
+
+locals {
+  tags = {
+    module  = "application-gateway"
+    example = "advanced"
+    usage   = "demo"
+  }
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "awg-rg"
+  location = "uksouth"
+}
```

Comment:
if new, add tags to resource group?

Reply:
The specifics of the creation of the resource group are not relevant to the example demonstrating the application gateway. I'd imagine it's unlikely that a user would copy the code for the resource group creation from the example of the app gateway. The resource group creation was only included for completeness.